public/plugin.tsx

```diff
@@ -110,13 +140,17 @@ export class AssistantPlugin
       },
       assistantEnabled,
       assistantActions,
+      registerPalantir: this.palantirRegistry!.register.bind(this.palantirRegistry),
+      getPalantir: this.palantirRegistry!.get.bind(this.palantirRegistry),
```

Comment:
Is this `!` necessary? Seems `this.palantirRegistry` is initialized in top of setup method. It's not undefined here.

Reply:
good catch will addrses.